.travis.yml

```diff
@@ -4,4 +4,4 @@ sudo: required
 dist: trusty
 before_install: ./maintainers/scripts/travis-nox-review-pr.sh nix
 install: ./maintainers/scripts/travis-nox-review-pr.sh nox
-script: ./maintainers/scripts/travis-nox-review-pr.sh build
+script: travis_wait 119 ./maintainers/scripts/travis-nox-review-pr.sh build
```

Comment:
It also appears to print the entire log file after the command has run, so I imagine it will just crash the build later. I don't think it ever reduces the amount of logs printed.

https://github.com/travis-ci/travis-build/blob/a25da5b27daa6d32d6050a6dfd0a70117462b8da/lib/travis/build/templates/header.sh#L112-L149